src/index.js

```diff
@@ -994,14 +942,7 @@ const updateFormElements = () => {
     }
     clearDisplayElements();
   }
-  if (
-    isWalletConnectConnected &&
-    activeProviderNameResult.innerText === 'wallet-connect'
-  ) {
-    for (const button of walletConnectButtons) {
```

Comment:
we don't have wallet connect specific buttons anymore since the previous issue was fixed, so we can remove this logic